.changelogs/8122.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Fixed column resizing when formulas results has changed width",
+  "type": "fixed",
+  "issue": 8122,
```

Comment:
Should be issue number. Issue #8061 we're solving here is a regression on develop, was never released so from the client point of view having this in changelog will be confusing.

Reply:
ok, I remove it.